.github/workflows/test.yml

```
@@ -64,6 +64,29 @@ jobs:
       - name: Run tests
         run: PYTHONPATH='.' pytest
 
+  stubtest:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: ['3.9']
+    steps:
+      - uses: actions/checkout@v3
+      - name: Setup system dependencies
+        run: |
+          sudo apt-get update
+          sudo apt-get install binutils libproj-dev gdal-bin
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v4
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Install dependencies
+        run: |
+          pip install -U pip setuptools wheel
+          pip install -r ./requirements.txt
+
+      - name: Run stubtest
+        run: bash ./scripts/stubtest.sh
```

Make the script executable?

```suggestion
        run: ./scripts/stubtest.sh
```

---

I prefer to use explicit `bash` here, because there are *multiple* problems with shebangs / permissions / etc. It is just not worth it :)

(I have a long and frightful story maintaining pure bash project 😅 https://github.com/sobolevn/git-secret/)